lumberjack.go

```diff
@@ -165,6 +170,14 @@ func (l *Logger) Write(p []byte) (n int, err error) {
 func (l *Logger) Close() error {
 	l.mu.Lock()
 	defer l.mu.Unlock()
+
+	if l.millCh != nil {
+		close(l.millCh)
```

Comment:
this is the actual fix for the leak, right? (just making sure I grok it)

Reply:
Yes